libs/sdk-ui-filters/src/AttributeFilter@next/hooks/useAttributeFilterControllerData.ts

```diff
@@ -70,6 +71,8 @@ export function useAttributeFilterControllerData(
     const isFiltering = useIsFiltering(handler);
 
     const parentFilterAttributes = handler.getLimitingAttributeFiltersAttributes();
+    const displayForms = attribute ? attribute.displayForms : [];
```

Comment:
Detail: should this be 
```
const displayForms = attribute?.displayForms ?? [];
```
that way you could avoid the
```
displayForms: displayForms ?? []
```
and just do `displayForms`